auth/utils.py

```diff
@@ -0,0 +1,15 @@
+import jwt
+
+SECRET_KEY = '1234567890'  # TODO: Move to environment variables
+
+def is_valid_password(password):
+    return len(password) > 5
+
+def authenticate(u, p):
+    if u == "demo@example.com" and p == "password123":
+        return {"email": u}
+    return None
```

Comment:
chechk if password is password456
```suggestion
def authenticate(u, p):
    if u == "demo@example.com" and p == "password456":
        return {"email": u}
    return None
```